cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/BaseWidget.kt

```diff
@@ -56,25 +56,25 @@ abstract class BaseWidget(
      * Use this method for sending widget updated to user.
      * This method will add some required fields for you: `id`, `eventType`
      */
-    override fun send(state: JsonObject) {
-        state.put(CC.PROP_ID, id)
-        state.put(CC.PROP_EVENT_TYPE, PROP_EVENT_TYPE_WIDGET_UPDATE)
-        state.getJsonObject(CC.PROP_CONTENT).attachUpdateDate()
-        boardService.saveContent(id, state.getJsonObject(CC.PROP_CONTENT))
-        vertx.eventBus().send(CC.EVENT_SEND_MESSAGE_TO_WEBSOCKET, state)
+    override fun send(state: Any, dontWrap: Boolean) {
+        val data = if (dontWrap && state is JsonObject) state
```

Comment:
`send` method will now wrap data that needs to be sent in JsonObject as `content` property.
No need to do the same in all the widgets...